osquery/tables/system/python_packages.cpp

```diff
@@ -49,6 +51,11 @@ const std::set<std::string> kDarwinPythonPath = {
 };
 // clang-format on
 
+const std::set<std::string> kUserDirectoryPaths = {
+    ".pyenv/versions",
+    "Library/Python",
```

Comment:
I think the `~/.pyenv` would apply on Linux at least, but `~/Library` is probably macOS only. I wonder if we should make that one platform-specific?